raytracing/imagingpath.py

```diff
@@ -423,50 +433,62 @@ def save(self, filepath,
 
     def drawObject(self, axes):  # pragma: no cover
         """ Draw the object as defined by objectPosition, objectHeight """
-        (xScaling, yScaling) = self.axesToDataScaling(axes)
```

Comment:
Not sure why `self.axesToDataScaling` is gone because it is not clear `axesScale()`refers to scaling with respect to what?  Since the drawing functions are in data space (i.e. they use the coordinates of the plot), then it sounds natural to keep it in the name.
Scaling in graphs can become fairly complicated very quickly with the many reference frames (window, figure, axes, bounds, data, normalized, etc...), I would prefer if we kept the source-destination frames of reference explicit in the functions.

Reply:
We can keep the name axesToDataScaling. But to be clear, the function simply returns the dimensions of the display in data units. Also, since it's an axes property, i'm not sure that it should be defined in Matrix.

Reply:
So then the name is fine: we go from normalized axes coordinates to data coordinates. I don't remember all the details, but I believe there is a reference frame in normalized axes coordinates.